packages/expo-image/ios/EXImage.podspec

```diff
@@ -20,7 +20,7 @@ if using_custom_sd_web_image_webp_coder_version
 end
 
 Pod::Spec.new do |s|
-  s.name           = 'expo-image'
+  s.name           = 'EXImage'
```

Comment:
We will do another rename to `ExpoImage` when I rewrite it to Swift 😄 For now I'm fine with `EXImage`.

Reply:
it sounds good. there are too many EX prefixed stuffs, please help to do the rename together during the swift rewrite.